utilities/src/main/java/io/syndesis/qe/utils/OpenShiftUtils.java

```diff
@@ -58,19 +63,19 @@ public static NamespacedOpenShiftClient client() {
 
     public static Route createRestRoute(String openShiftNamespace, String urlSuffix) {
         final Route route = new RouteBuilder()
-                    .withNewMetadata()
```

Comment:
this was nicer and more readable with the spaces, please remove the change for this whole route definition